src/app/settings.ts

```diff
@@ -31,155 +32,156 @@ export class Settings {
   OriginalLocation: string = null;
   LevelName: string = null;
   Level_ID: number = null;
-  TemperatureType: string = "C";
-  TemperatureSymbol: string = "\u2103";
+  TemperatureType: 'C';
+  TemperatureSymbol: '\u2103';
   DatabaseName: string = null;
-  DataUnits: string = "";
```

Comment:
Just because the linter says something is "wrong" doesn't mean it always is. I couldn't get the code to compile because of an issue here.
`DataUnits: '';` should be changed back to the original `DataUnits: string = '';

I get the following error if it's not specified as a string when I try to build the project:
```
ERROR in /home/mrsharky/Code/climate2/src/app/helpers.ts (277,10): Type '"Kelvins"' is not comparable to type '""'.

ERROR in /home/mrsharky/Code/climate2/src/app/helpers.ts (278,10): Type '"degK"' is not comparable to type '""'.
```